vagrant.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -euo pipefail
+_FINISH(){
+    RESULT=($?)
+    if [ ${RESULT} -eq 0 ]; then
+        echo ""
+        echo "Tests completed normally..."
+        vagrant destroy ${BOX}
+        echo ""
+    else
+        echo ""
+        echo "** Test failed.  Leaving '${BOX}' running for debug." 2>&1 | tee -a ${tee_file}
+        echo "** Be sure to halt or destroy prior to re-running the check" 2>&1 | tee -a ${tee_file}
+        echo "** Logs are stored at ${tee_file}"
+        echo ""
+        exit 1
+    fi
+}
+
+# When make calls bash, the real signals are not surfaced
+# correctly to trap.  So we trap on EXIT and then sort it
+# out in _FINISH
+trap _FINISH EXIT
+
+BOXES="fedora_atomic centos_atomic"
+
+is_running() {
+    status=$(vagrant status | grep ${BOX} | awk '{print $2}')
+    if [ ${status} == "running" ]; then
+        echo ""
+        echo "'${BOX}' is already running.  Perform a 'vagrant halt|destroy ${BOX}'"
```

Comment:
@jlebon OK I have implemented your suggestion but I hit ...

```*** 'fedora_atomic' is already running.  Re-syncing and rerunning test ...

/usr/share/gems/gems/net-ssh-2.9.1/lib/net/ssh/transport/session.rb:67:in `initialize': Object#timeout is deprecated, use Timeout.timeout instead.
/usr/share/gems/gems/net-ssh-2.9.1/lib/net/ssh/transport/session.rb:84:in `initialize': Object#timeout is deprecated, use Timeout.timeout instead.
==> fedora_atomic: Rsyncing folder: /home/bbaude/docker/atomic/ => /home/vagrant/atomic
==> fedora_atomic:   - Exclude: [".vagrant/", ".tmp*"]
+ mount --make-rshared /
+ systemctl start docker
+ '[' -f /run/ostree-booted ']'
+ ostree admin unlock
error: Deployment is already in unlocked state: development
Connection to 192.168.121.217 closed.

** Test failed.  Leaving 'fedora_atomic' running for debug.
** Be sure to halt or destroy prior to re-running the check
** Logs are stored at fedora_atomic_2016_11_03_09_34.log

Makefile:97: recipe for target 'vagrant-check' failed
make: *** [vagrant-check] Error 1
```

So can your rh ci script handle this?